README.md

```diff
@@ -7,8 +7,8 @@ http://nsq.io/
 Offline documentation requires [jekyll][jekyll]:
 
 ```bash
-$ gem install jekyll
-$ jekyll serve --safe --baseurl ''
+$ bundle install
+$ bundle exec jekyll serve --safe --watch --baseurl ''
```

Comment:
Jekyll is running for me now, but when i visit http://0.0.0.0:4000/ it's redirecting to http://nsq.io//